week-10/userApi/.vite/deps/_metadata.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
don't think, we need this .vite configuration, can you tell me why we have this?